remixer_example/src/main/java/com/google/android/apps/remixer/RemixerApplication.java

```diff
@@ -35,5 +36,7 @@
   public void onCreate() {
     super.onCreate();
     RemixerInitialization.initRemixer(this);
+    Remixer.getInstance().setSynchronizationMechanism(
+        new LocalStorage(getApplicationContext(), "Remixer"));
```

Comment:
Once we start writing to disk, maybe it should be up to the developer. I think offering local value syncing (without saving) is a good default.

Also since it uses android-specific stuff it has to be in an android library (remixer_storage) instead of a java library (remixer_core), so it would have to be in RemixerInitialization. Even so **I still argue against it** since that initialization is just the barebones so Remixer **works** correctly (not leaking, types loaded). 

I could add more documentation instead if you want (readme?)